spacy_llm/tasks/lemma.py

```diff
@@ -90,6 +95,11 @@ def generate_prompts(self, docs: Iterable[Doc]) -> Iterable[str]:
 
     def parse_responses(
         self, docs: Iterable[Doc], responses: Iterable[str]
+    ) -> Iterable[Doc]:
+        return self._parse_responses(docs, responses)
+
+    def _parse_responses_default(
```

Comment:
This would be temporary. If we decide to do this, all response parse functions would be registered.